deployment/database/liquibase/data_textures.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:mongodb="http://www.liquibase.org/xml/ns/mongodb"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+         http://www.liquibase.org/xml/ns/mongodb
+         http://www.liquibase.org/xml/ns/mongodb/liquibase-mongodb-latest.xsd">
+
+  <changeSet id="create_data_textures_collection" author="ssteinig">
+    <mongodb:createCollection collectionName="data_textures"/>
+    <mongodb:createIndex collectionName="data_textures">
+      <mongodb:keys>
+        { forecast_base_time: 1, tim_start: 1, tim_end: 1, texture_uri: 1, source: 1 }
+      </mongodb:keys>
+      <mongodb:options>
+        {unique: true, name: "uniq_texture_idx"}
+      </mongodb:options>
+    </mongodb:createIndex>
+  </changeSet>
+</databaseChangeLog>
```

Comment:
I think Git is complaining about not having a newline at the end

Reply:
added a new line, but there is also none for the other xml files like `in_situd_data.xml`, don't know whether necessary or not